Helper/Requests.php

```diff
@@ -85,69 +101,53 @@ public function buildCustomerData(
     ) {
         if ($customerId > 0) {
             $request['shopperReference'] = $customerId;
+
+            $customer = $this->customerRepository->getById($customerId);
+
+            if (empty($request[self::DATE_OF_BIRTH]) && $dateOfBirth = $customer->getDob()) {
+                $request[self::DATE_OF_BIRTH] = $dateOfBirth;
+            }
+
+            //TODO check if Gender can be retrieved
```

Comment:
is the todo still applicable?

Reply:
not for this ticket but for future improvements